stickyuploads/tests/test_views.py

```diff
@@ -33,13 +33,16 @@ def test_upload(self):
             response = self.client.post(self.url, data)
         self.assertEqual(response.status_code, 200)
         result = json.loads(response.content.decode('utf-8'))
-        storage = TempFileSystemStorage()
         filename = os.path.basename(self.temp_name)
+        # We can't test full filename value because there are random letters
+        self.assertIn('filename', result)
+        self.assertIn(filename, result['filename'])
+        self.assertIn('stored', result)
         expected = {
             'is_valid': True,
-            'filename': filename,
+            'filename': result['filename'],
```

Comment:
This expected value is going to be compared against the `result` which is more or less a tautology at this point. You can mock the name creation to assert the expected name without pulling it from the results.